packages/api/utils/errors.ts

```diff
@@ -5,6 +5,15 @@ import { ErrorStatusCode, OtherErrorResponse } from "./response";
  */
 export const DATABASE_ERROR = new OtherErrorResponse("Database error", ErrorStatusCode.INTERNAL_SERVER_ERROR);
 
+/**
+ * A "last-resort" error, to be used when it is unknown what caused a particular
+ * error (or to entirely mask error details)
+ */
+export const UNKNOWN_ERROR = new OtherErrorResponse(
+  "An unexpected error occurred",
+  ErrorStatusCode.INTERNAL_SERVER_ERROR
+);
+
```

Comment:
We shouldn't be using this since everything that would should actually use it should `throw`.

Reply:
This is true, removing.